test/functional/sessions_controller_test.rb

```diff
@@ -5,7 +5,6 @@ class SessionsControllerTest < ActionController::TestCase
     setup do
       @user = User.new(email_confirmed: true, handle: "test")
       @user.enable_mfa!(ROTP::Base32.random_base32, :ui_only)
-      @request.cookies[:mfa_feature] = "true"
```

Comment:
This isn't strictly relevant to this change, I just noticed that this unused flag was still kicking around in tests and thought this a prudent time to remove it.